src/main/java/org/ice4j/attribute/UsernameAttribute.java

```diff
@@ -56,8 +76,17 @@
      *   offset is equal to the index of the first byte after length)
      * @param length the length of the binary array.
      */
+    @Override
     void decodeAttributeBody(byte[] attributeValue, char offset, char length)
     {
+        if (stripTrailingZeroes)
+        {
+            while (length > 0 && attributeValue[offset + length] == 0)
```

Comment:
a nitpick i guess, but given there's no minimum length, i'm wondering if we should prevent this from going to 0 length?  it seems arbitrary to stop it at length 1.